help.md

```diff
@@ -0,0 +1,82 @@
+% NEXUS(1) Container Image Pages
+% Sonatype
+% November 21, 2016
+
+# NAME
+nexus \- Nexus Repository Manager container image
+
+# DESCRIPTION
+The nexus image provides a containerized packaging of the Nexus Repository Manager.
+Nexus Repository Manager is a repository manager with universal support for popular component formats including Maven, Docker, NuGet, npm, PyPi, Bower and more.
+
+The nexus image is designed to be run by the atomic command with one of these options:
+
+`run`
+
+Starts the installed container with selected privileges to the host.
+
+`stop`
+
+Stops the installed container
+
+`uninstall`
+
+Removes the installed container, not the image
+
+The container itself consists of:
+    - RHEL7 base image
```

Comment:
If we can provide help for the CentOS version then this should indicate it could be either.